examples/visualization/styles/helpers/size-bins-style.html

```diff
@@ -52,26 +52,32 @@
           breaks: [0, 100, 200, 300, 365],
           sizes: [5, 20, 50, 75]
         });
-
-        const airbnbLayer = new carto.viz.Layer(
-          'listings_madrid',
-          availabilityStyle,
-          {
-            mapOptions: {
-              bufferSize: {
-                mvt: 30
-              }
+        const airbnbSource = new carto.viz.CARTOSource('listings_madrid', {
+          mapOptions: {
```

Comment:
It's nice that the source allows passing parameters (especially for advanced users), but if Layer doesn't render well without them, I guess we should hide that logic somehow.

Another quetion, why 30? is that pixels?

Reply:
Yup, according to the [specification](https://github.com/CartoDB/Windshaft/blob/master/doc/MapConfig-1.8.0.md) that size is in pixels. 

I agree, we should add default values in order to render well the layers. However, we only can add a default value for lines ATM. For points we have to wait for the fix https://github.com/visgl/deck.gl/pull/4557

Reply:
It's gonna be difficult to get different defaults from different geometry types, as these params goes on maps instantation and right now we get the geometry type precisely as the response of that instantiation.